wallet/wallet.go

```diff
@@ -3788,6 +3807,34 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 			backendError: err,
 		}
 
+	// Returned by bitcoind when a transaction does not meet the fee
+	// requirements to be accepted into mempool. This happens when the
+	// mempool reached its limits and is now purging low fee transactions.
+	// Increasing the mempool size limit (maxmempool) will solve this
+	// problem so we log the appropriate bitcoind config setting.
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L510
+	case match(err, "mempool min fee not met"):
+		returnErr = &ErrMempoolFee{
+			backendError: err,
+		}
+		log.Infof("%v: tx not accepted into mempool because of low "+
+			"fees, consider raising your bitcoind maxmempool "+
+			"config value", txid)
+
+	// Returned by bitcoind when a transaction does not meet the fee
+	// requirements to be accepted into mempool because the policy of the
+	// mempool has a higher min relay fee. Default value for bitcoind is
+	// 1000 sat/kvbyte but this is configurable so we log the appropriate
+	// bitcoind config setting.
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L514
+	case match(err, "min relay fee not met"):
```

Comment:
Do we also want to attempt to catch the various RBF related errors? So eg: replacement transaction's fee isn't high enough.

Reply:
ok I see its already remapped to `ErrDoubleSpend`: https://github.com/lightningnetwork/lnd/blob/master/lnwallet/btcwallet/btcwallet.go#L1168